Vector angularVelocity
Vector linearAcceleration
Quaternion orientation
uint64 timeStamp